include/swift/SIL/SILLocation.h

```diff
@@ -755,17 +755,18 @@ class SILDebugScope;
 
 /// A SILLocation paired with a SILDebugScope.
 class SILDebugLocation {
-  const SILDebugScope *Scope = nullptr;
-  SILLocation Location;
+  const SILDebugScope *debugScope;
+  SILLocation location;
 
 public:
   SILDebugLocation()
-      : Scope(nullptr),
-        Location(RegularLocation::getAutoGeneratedLocation()) {}
-  SILDebugLocation(SILLocation Loc, const SILDebugScope *DS)
-      : Scope(DS), Location(Loc) {}
-  SILLocation getLocation() const { return Location; }
-  const SILDebugScope *getScope() const { return Scope; }
+      : debugScope(nullptr),
```

Comment:
I prefer the = nullptr spelling, since it makes it harder to forget to initialize the member when adding a new constructor.

Reply:
Ok. To me it just looked redundant. I am fine fixing it in a follow on commit.